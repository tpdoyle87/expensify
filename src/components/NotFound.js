import React from 'react'
import { Link } from 'react-router-dom'

const NotFound = () => (
    <div>
      <p>404</p>
      <Link to="/">Go Home</Link>
    </div>
)

export default NotFound
