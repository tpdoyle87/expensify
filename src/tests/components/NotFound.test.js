import NotFound from '../../components/NotFound'
import React from 'react'
import { shallow } from 'enzyme'

test('it should render the not found page', () => {
  const wrapper = shallow(<NotFound />)
  expect(wrapper).toMatchSnapshot();
})
