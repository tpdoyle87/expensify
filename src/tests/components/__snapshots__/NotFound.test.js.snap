// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render the not found page 1`] = `
<div>
  <p>
    404
  </p>
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
